README.md

```diff
@@ -61,9 +61,42 @@ A few elements of the API are visible in this example:
   - The object passed into the constructor, if it’s still available.
   - `undefined`, if nothing else was pointing to the object and it was already garbage-collected.
 
+### Weak references and the event loop
+
+When you call `deref()` on a weak reference and you get back a value---i.e., not `undefined`---that value is protected from garbage collection for the rest of the turn ([Job](https://tc39.github.io/ecma262/#sec-jobs-and-job-queues)).  Therefore this function will always return true:
+
+```js
+function test(x) {
+  const ref = new WeakRef(x)
+  const y = ref.deref();
+  const z = ref.deref();
+  return y === z;
+}
+```
+
+However this function may return true or false:
+
+```js
+async function test(x) {
+  const ref = new WeakRef(x)
+  const y = ref.deref();
+  await new Promise(resolve => setTimeout(resolve));
+  const z = ref.deref();
```

Comment:
This example is wrong. `y` is a strong pointer to the object. While `y` is live, the object it cannot be collected. Therefore, derefing a weakref to it, even in a later turn, must give back the object, so long as `y` is live. Since `y` and `z` both participate in the comparison, clearly `y` is still live at that point.